### PR TITLE
Refurbished the invite button in ContactsFragment #8

### PR DESCRIPTION
**Pull request by boorah**

### CODE REVIEW

**Comment by Bhavnaharitsa** [comment]
Please post a screen shot of the changes

**Comment by boorah** [comment]
Screenshot_20191220-193332
Screenshot_2019-12-20-19-35-05-975_com example chatup

File: app/build.gradle [hunk 1]
```diff
@@ -77,4 +77,8 @@ dependencies {
     implementation 'com.android.support:recyclerview-v7:28.0.0'
     implementation 'com.github.AppIntro:AppIntro:5.1.0'
     implementation 'com.github.stfalcon:chatkit:0.3.3'
+    implementation 'com.google.firebase:firebase-analytics:17.2.0'
```
Comment (Bhavnaharitsa) [inline app/build.gradle]: why this dependency?

File: build.gradle [hunk 2]
```diff
@@ -9,6 +9,7 @@ buildscript {
     dependencies {
         classpath 'com.android.tools.build:gradle:3.4.1'
         classpath 'com.google.gms:google-services:4.2.0'
+        classpath 'com.google.gms:google-services:4.3.2'
```
Comment (Bhavnaharitsa) [inline build.gradle]: why this too?

File: app/build.gradle (hunk 1 above)
Comment (boorah) [reply app/build.gradle]: I am new to firebase, so I followed the tutorial in the website

File: build.gradle (hunk 2 above)
Comment (boorah) [reply build.gradle]: Same here

File: app/build.gradle (hunk 1 above)
Comment (Bhavnaharitsa) [reply app/build.gradle]: remove that..not necessary

File: build.gradle (hunk 2 above)
Comment (Bhavnaharitsa) [reply build.gradle]: remove

File: app/build.gradle (hunk 1 above)
Comment (boorah) [reply app/build.gradle]: Okay

File: build.gradle (hunk 2 above)
Comment (boorah) [reply build.gradle]: So I have to close this request and again create a new one?

**Comment by boorah** [comment]
Now?

**Pull request merged**